pipeline/shared/duplicati.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+. "$( cd "$(dirname "$0")" ; pwd -P )/../docker-run/error_handling.sh"
+
+export DUPLICATI_ROOT="/application/"
```

Comment:
I see a few references to `/application` in the other scripts. Was the intent to reference `DUPLICATI_ROOT` if possible?

Reply:
Okay I need to check out how the entire logic was again :) The /application is the mount point within the docker container, where things are actually happening. I think this was especially relevant to the packaging of rpm's, deb's etc. The duplicati.sh script should be important for things that run within the container.

Reply:
There was one occurrence that I changed now. As said, the duplicati.sh script is for the job files in the stages subdirs.